Practice1/palindrome

```diff
@@ -0,0 +1,36 @@
+def isPalindrome n
```

Comment:
You need to review this files they are not *.rb files and you need to remove the + chars